src/main/webapp/app/overview/exercise-details/exercise-action-button.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, HostBinding, Input } from '@angular/core';
 
 @Component({
-    selector: '[jhi-exercise-action-button]',
+    /* tslint:disable-next-line component-selector */
```

Comment:
I added the ts lint ingore because our configuration ignores these types of components. is used them based on https://angular.io/guide/styleguide#style-05-03 . similar to the mat-button